force-app/main/default/classes/LeadTriggerHandler.cls

```diff
@@ -29,18 +44,21 @@ public with sharing class LeadTriggerHandler {
 	 *   change the title to 'Assistant'.
 	 */
 	public static void handleTitleNormalization(List<Lead> leadsToNormalize) {
+
 		for (Lead ld : leadsToNormalize) {
-			if (ld.title == 'vp' || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
+			if (ld.title == null) {
+				continue;
+            } else if (ld.title.contains('vp') || ld.title.contains('v.p.') || ld.title.contains('vice president')) {
```

Comment:
Try using a list of words you are looking for then that you want to catch for the title then checking if the title is in the list.